src/renderer/views/Post/Post.js

```diff
@@ -35,7 +35,7 @@ export default defineComponent({
     }
   },
   watch: {
-    async $route() {
+    async '$route.params.id'() {
```

Comment:
I'm aware this is so the route watch doesnt get triggered when an authorid is added but is there any performance benefit to this as well? Just wondering if we should use this in our other views

Reply:
The Vue Router docs do recommend watching only what you need to watch. We can definitely look into doing it in other views too, the hashtag view is an obvious one because that has a single param and no query parameters, but other views like the Search one might be more difficult to narrow down the watcher, as that has a parameter and lots of query parameters.

> The route object is a reactive object. In most scenarios, you should avoid watching the whole route object. Instead, you can directly watch the properties you are expecting to change

https://router.vuejs.org/guide/advanced/composition-api.html#Accessing-the-Router-and-current-Route-inside-setup